app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -68,13 +68,14 @@
 
           <TextView
             android:id="@+id/chapter_summary"
-            android:layout_width="0dp"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginStart="8dp"
             android:layout_marginTop="8dp"
             android:layout_marginEnd="8dp"
             android:layout_marginBottom="4dp"
             android:fontFamily="sans-serif"
+            android:textAlignment="viewStart"
```

Comment:
@BenHenning Can you confirm if we can use this attribute or not?

Reply:
Seems fine, though I'm unsure when we should be using ``textStart`` vs. ``viewStart``. @veena14cs does ``textStart`` have the same behavior? Per [documentation](https://developer.android.com/reference/android/view/View#attr_android:textAlignment) it may be better to always default to ``textStart`` since it's more likely to look correct (since it provides alignment based on the paragraph rather than the view), while also accounting for RTL scenarios.

Reply:
Here, what I seem to be reasonable is, we are aligning based on the layout direction and if we want based on textdirection we need to add direction for paragraph. As it does not understand if the paragraph is LTR or RTL.

Reply:
I think `viewStart` is correct for this. http://androidsbs.blogspot.com/2013/11/androidtextalignment.html#:~:text=textStart%20%2D%20Align%20to%20the%20start%20of%20the%20paragraph%2C%20e.g.%20ALIGN_NORMAL.&text=center%20%2D%20Center%20the%20paragraph%2C%20e.g.,is%20LTR%2C%20and%20ALIGN_RIGHT%20otherwise.

Reply:
Thanks for this refernce @rt4914 .

Reply:
Oh, I assumed that the paragraph's start with align with RTL/LTR. If that's not the case, that's a bit weird.

Either way, I think this approach is sensible.

Reply:
Thanks Ben!